library/blob.c

```diff
@@ -1057,6 +1053,83 @@ static int eblob_blob_iterator(struct eblob_iterate_priv *iter_priv)
 	return err;
 }
 
+
+/**
+ * eblob_disk_control_cmp_by_position() - compares dc's by position in a blob file
+ */
+static int eblob_disk_control_cmp_by_position(const void *lhs, const void *rhs) {
+	const struct eblob_disk_control *left = (const struct eblob_disk_control *)lhs;
+	const struct eblob_disk_control *right = (const struct eblob_disk_control *)rhs;
+	if (left->position == right->position)
```

Comment:
This function will be called many times, so I would start with more common cases, e.g.
```
if (left->position > right->position) {
    return 1;
} else if (left->position < right->position) {
    return -1;
} else {
    return 0;
}
```

Reply:
done with slight different order